changelog.yaml

```
@@ -16,7 +16,24 @@
 
 - version: "NEXT"
   date: TBD
-  
+  changes:
+  - type: enhancement
+    impact: minor

```

https://github.com/SAP/stewardci-core/pull/162#discussion_r511940085
```suggestion
    impact: major
```